spec/decorators/pagination_decorator_spec.rb

```diff
@@ -5,22 +5,41 @@
     described_class.new(content)
   end
 
-  let(:mod) { Training::Module.by_name(:alpha) }
-  let(:content) { mod.page_by_name('1-1-1') }
+  context('when First Submodule') do
```

Comment:
Can we chat about this please; changing the context to what is being tested will help.

`when feedback questions are skipped`